po/common/fr.po

```diff
@@ -6934,4 +6934,36 @@ msgstr "Statut vérifié"
 
 msgctxt "creation_date"
 msgid "Date de création"
-msgstr "Date de création"
\ No newline at end of file
+msgstr "Date de création"
+
+msgctxt "enter_main_contact_username"
+msgid "Nom du contact principal :"
```

Comment:
the msgid should be the English text, otherwise it won't work
(same in strings below)